controllers/auth.js

```diff
@@ -16,20 +17,50 @@ exports.authPost = async (req, res) => {
         if (!isPassValid) {
             return res.status(400).json({ message: "Invalid password" });
         }
-        const token = jwt.sign({ id: user.id }, config.get("secretKey"), {
-            expiresIn: "1h",
-        });
-        return res.status(200).send({
-            token,
-            user: {
-                id_user: user.id,
-                name: user.name,
-                userRole: user.userRole,
-                photo: user.photo,
-            },
-        });
+
+        updateTokens(user.id).then((tokens) =>
+            res.status(200).send({
+                tokens,
+                user: {
+                    id_user: user.id,
+                    name: user.name,
+                    userRole: user.userRole,
+                    photo: user.photo,
+                },
+            })
+        );
     } catch (e) {
         console.log(e);
         res.send({ message: "Server error" });
     }
 };
+
+exports.refreshTokens = (refreshToken) => {
+    let payload = jwt.verify(refreshToken, config.get("secretKey"));
+
+    try {
+        if (payload.type !== "refresh") {
+            res.status(400).json({ message: "Invalid Token!" });
+            return;
+        }
+    } catch (e) {
+        if (e instanceof jwt.TokenExpiredError) {
+            res.status(400).json({ message: "Token expired!" });
+            return;
+        } else if (e instanceof jwt.JsonWebTokenError) {
+            res.status(400).json({ message: "Invalid Token!" });
```

Comment:
no need to send message about tokens to the client

Reply:
changed to err 500 and message: "Server Error" or "Unauthorized"